staging/vhost-device-gpu/src/vhu_gpu.rs

```diff
@@ -166,8 +167,9 @@ impl VhostUserGpuBackend {
                 virtio_gpu.resource_create_3d(resource_id, resource_create_3d)
             }
             GpuCommand::ResourceUnref(info) => virtio_gpu.unref_resource(info.resource_id),
-            GpuCommand::SetScanout(_info) => {
-                panic!("virtio_gpu: GpuCommand::SetScanout unimplemented");
+            GpuCommand::SetScanout(info) => {
+                debug!("SetScanout: {info:?}");
+                virtio_gpu.set_scanout(info.scanout_id, info.resource_id, None)
```

Comment:
Here `None` is passed as the `scanout_data` argument. But info contains things (namely a `virtio_gpu_rect` that has `width`/`height`) that should probably be stored with the resource.

Reply:
well for crosvm i see that the pass none for `scanout_data` and pass something for set_scanout in SetScanout, so which means that info should not contain any data in width or height which is ``` SetScanout: virtio_gpu_set_scanout { r: virtio_gpu_rect { x: 0, y: 0, width: 640, height: 480 }, scanout_id: 0, resource_id: 0 } ```

Reply:
Hmm, interesting. I was looking at the C vhost-user-gpu and there they store the width and height. Not sure if we need to store it, I guess we will see if need it later. But we should definetly send the `width`/`height` to QEMU using the `GpuBackaned`.